frontend/src/components/AgencyProfile/DeleteModal.jsx

```diff
@@ -20,14 +21,31 @@ function DeleteModal({ showModal, toggleModal, agencyName, agencyNumber, agencyI
    * Function deletes the given agency and returns the user back to the agency page
    */
   function deleteAgency() {
-    fetch(`http://localhost:8000/agency/${agencyId}`, { method: "DELETE" })
+    fetch(`http://localhost:8000/agency/${agencyId}`, {
```

Comment:
might want to use an environment variable instead

Reply:
It's ok to leave this bc I'm gonna update this for deployment right after you merge this PR in anyways